kDrive/AppDelegate.swift

```diff
@@ -175,16 +146,6 @@ final class AppDelegate: UIResponder, UIApplicationDelegate, AccountManagerDeleg
         Log.appDelegate("Unable to register for remote notifications: \(error.localizedDescription)", level: .error)
     }
 
-    func applicationDidEnterBackground(_ application: UIApplication) {
-        Log.appDelegate("applicationDidEnterBackground")
-        backgroundTasksService.scheduleBackgroundRefresh()
-
-        if UserDefaults.shared.isAppLockEnabled,
-           !(window?.rootViewController?.isKind(of: LockedAppViewController.self) ?? false) {
-            lockHelper.setTime()
-        }
-    }
-
     func application(
         _ application: UIApplication,
         performActionFor shortcutItem: UIApplicationShortcutItem,
```

Comment:
As discussed in DM, shortcuts are now processed by the scene

Reply:
Good catch, migrated actions in https://github.com/Infomaniak/ios-kDrive/pull/1206